src/win/dtwin.c

```diff
@@ -338,14 +338,15 @@ void dtwin_set_thread_name(DWORD dwThreadID, const char *threadName)
   info.dwThreadID = dwThreadID;
   info.dwFlags = 0;
 
-  // Yes, don't get heart attack, naming of thread is done by raising a special exception on Windows
-  // https://msdn.microsoft.com/en-us/library/xcb2z8hs(v=vs.71).aspx
+  // We set the thread name by raising an exception.
+  // Regarding the alternative method and their comparison, see:
+  // https://learn.microsoft.com/en-us/visualstudio/debugger/how-to-set-a-thread-name-in-native-code
   RaiseException(MS_VC_EXCEPTION, 0, sizeof(info) / sizeof(DWORD), (const ULONG_PTR *)&info);
 }
 
-// This is taken from: https://git.gnome.org/browse/glib/tree/gio/glocalfile.c#n2269
-// The glib version of this function unfortunately shows always confirmation dialog boxes
-// This version does thrashing silently, without dialog boxes: FOF_SILENT | FOF_NOCONFIRMATION
+// This is taken from: https://github.com/GNOME/glib/blob/main/gio/glocalfile.c#L2357
```

Comment:
It should be https://gitlab.gnome.org/GNOME/glib/blob/main/gio/glocalfile.c#L2357

Reply:
I agree, you're right, it is better to specify the original and not the mirror.